components/multi-input/multi_input.js

```diff
@@ -15,7 +11,7 @@ Template.InternationalPhoneMultiInput.helpers({
     var exampleNumber = Phoneformat.exampleMobileNumber(countryCode);
     var intlNumber = Phoneformat.formatInternational(countryCode, exampleNumber);
 
-    return intlNumber.replace(/[0-9]/g, 'X');
+    return intlNumber.replace(/[0-9]/g, 'x');
```

Comment:
Does this really look better? When you start including `-` or `( )` it seems to look strange.